rotkehlchen/tests/api/test_history_events_export.py

```diff
@@ -121,6 +122,31 @@ def test_history_export_download_csv(
     assert_csv_export_response(response, download_dir, is_download=True)
 
 
+@pytest.mark.vcr
```

Comment:
Why do you need to VCR this? Does it have any remote queries?

Reply:
When it does the `exporthistoryeventresource` api call there are some historical price queries to cryptocompare.

Reply:
I've removed the line that disabled mocking price queries and removed the VCR per [yabir's suggestion in discord](https://discord.com/channels/657906918408585217/1080783409640648765/1281666453698908241)